src/App.vue

```diff
@@ -1,23 +1,357 @@
-<script setup>
-import Header from './layout/Header.vue';
-import CveHeader from './components/CveHeader.vue';
-import CveOverview from './components/CveOverview.vue';
-import FaqSection from './components/FaqSection.vue';
-import CveFooter from './components/CveFooter.vue'
-import Footer from './layout/Footer.vue'
-import CveScoreDetails from './components/CveScoreDetails.vue'
-</script>
-
 <template>
   <Header />
   <CveHeader />
   <CveOverview />
-  <CveScoreDetails/>
+    <cp-affected-packages-table class="affected-packages-container" hoverable id="table"></cp-affected-packages-table>
+  <CveScoreDetails />
   <FaqSection />
   <CveFooter />
   <Footer />
 </template>
 
+<script setup>
+import Header from "./layout/Header.vue";
+import CveHeader from "./components/CveHeader.vue";
+import CveOverview from "./components/CveOverview.vue";
+import FaqSection from "./components/FaqSection.vue";
+import CveFooter from "./components/CveFooter.vue";
+import Footer from "./layout/Footer.vue";
+import CveScoreDetails from "./components/CveScoreDetails.vue";
+import "@cpelements/elements/cp-affected-packages-table/cp-affected-packages-table.js";
+import { onMounted } from "vue";
+
+const data = [
+  {
+    cpe: "cpe:/o:redhat:enterprise_linux:7",
+    product: "Red Hat Enterprise Linux 7",
+    advisory: {
+      name: "RHSA-2023:1333",
+      release_date: "2023-03-20T09:38:02+00:00",
+      type: "RHSA",
+      url: "https://access.redhat.com/errata/RHSA-2023:1333",
+    },
+    package: "firefox",
+    state: "Fixed",
+    lifecycle_url: "https://access.redhat.com/support/policy/updates/errata/",
+    impact: "critical",
```

Comment:
Data can be moved to another file and exported from there, so that all of the data is in one place